src/components/cylc/gscan/GScan.vue

```diff
@@ -373,12 +372,6 @@ export default {
     }
   },
   methods: {
-    getWorkflowClass (node) {
-      return {
-        'c-workflow-stopped': node && node.status && node.status === WorkflowState.STOPPED.name
-      }
-    },
-
```

Comment:
Not sure it was the best idea to move this out of GScan but no biggie.

Reply:
I agree but it was a quick solution, and the SCSS targeting the `c-workflow-stopped` class is nested inside the `c-gscan-workflow` class selector so it works without a problem (for now)